### PR TITLE
[Doc Improvement][groupchat in lowercase for defaultGroupCapability][3970899]

### PR DESCRIPTION
**Pull request by v-ramyas5**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/schema/manifest-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/39736ea381bb1f0a6642d0d88bf181a3fb6b07ac/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema.md) | 86 | 109 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b53434dc-3551-473f-bfb3-13682734aa20) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/resources/schema/manifest-schema.md [hunk 1]
```diff
@@ -814,7 +814,7 @@ When a group install scope is selected, it defines the default capability when t
 |Name| Type| Maximum size | Required | Description|
 |---|---|---|---|---|
 |`team`|String|||When the install scope selected is `team`, this field specifies the default capability available. Options: `tab`, `bot`, or `connector`.|
-|`groupChat`|String|||When the install scope selected is `groupChat`, this field specifies the default capability available. Options: `tab`, `bot`, or `connector`.|
```
Comment (Wajeed-msft) [inline msteams-platform/resources/schema/manifest-schema.md:817]: Group chat is reference in multiple place. Please fix it.

image

**Review by Wajeed-msft — CHANGES_REQUESTED** [review]

File: msteams-platform/resources/schema/manifest-schema.md (hunk 1 above)
Comment (v-ramyas5) [reply msteams-platform/resources/schema/manifest-schema.md:817]: Hi @Wajeed-msft 
In the current schema v1.17, groupchat is in lower case only for the property [`defaultGroupCapability`](https://teams.microsoft.com/l/message/19:9da4360914d24cf58a6a0c54af6c6211@thread.skype/1723532567238?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=32e3b156-66b2-4135-9aeb-73295a35a55b&parentMessageId=1718105311068&teamName=Teams%20Platform&channelName=Platform%20Doc-Queries&createdTime=1723532567238) while all the other instances are in camelcase. We have [requested the team](https://teams.microsoft.com/l/message/19:9da4360914d24cf58a6a0c54af6c6211@thread.skype/1726811029876?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=32e3b156-66b2-4135-9aeb-73295a35a55b&parentMessageId=1718105311068&teamName=Teams%20Platform&channelName=Platform%20Doc-Queries&createdTime=1726811029876) to fix it in the next schema releases. But we need to update the doc to support the current schema version.

**Review by Wajeed-msft — DISMISSED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/schema/manifest-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f2a0360aca289bfde481e013dbe5fccebef500b5/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema.md) | 86 | 108 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/36adde7c-fe39-4f66-ab09-5780b528d18b) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/schema/manifest-schema-dev-preview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/44aba52c81db4f12267e6794a123ac01bd5011aa/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema-dev-preview.md) | 85 | 130 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2267315d-440c-44c2-9f0e-e302773b0e61) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/schema/manifest-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/44aba52c81db4f12267e6794a123ac01bd5011aa/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema.md) | 86 | 108 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/cb868028-08e0-4774-b26c-677850ee3f6d) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/schema/manifest-schema-dev-preview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f1167954a70b5f2381b628511e5380bcef1e49af/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema-dev-preview.md) | 85 | 131 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c4bfe203-d669-47d7-83ce-99922ac78550) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/schema/manifest-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f1167954a70b5f2381b628511e5380bcef1e49af/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema.md) | 86 | 108 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0f0b821b-76c4-498c-b786-d2b20f66251a) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/schema/manifest-schema-dev-preview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/054c8556d685778a8ba694bcb9734d3652508d5c/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema-dev-preview.md) | 85 | 135 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/288d8381-dec0-47b7-9a8b-5ed6d8a298b9) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/schema/manifest-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/054c8556d685778a8ba694bcb9734d3652508d5c/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema.md) | 86 | 115 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2ab6babd-68e8-48ba-8a88-bb48877b4036) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/schema/manifest-schema-dev-preview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f74c3b8577fba3ea5e733f4884ae07ecb800682b/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema-dev-preview.md) | 85 | 135 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/4391124a-2829-4bce-b901-d3b98841c9e7) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/schema/manifest-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f74c3b8577fba3ea5e733f4884ae07ecb800682b/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema.md) | 86 | 115 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1d786f41-3dbe-42c6-8726-91148354884f) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by surbhigupta12 — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/schema/manifest-schema-dev-preview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/fb9fe27ea8cf6ad0261b56e33e476672e61a3815/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema-dev-preview.md) | 85 | 135 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/fba06958-df35-4deb-9ac0-168a2ae041ee) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/schema/manifest-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/fb9fe27ea8cf6ad0261b56e33e476672e61a3815/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema.md) | 86 | 115 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/4dd54c68-0b1a-4100-b409-564b505acb68) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Pull request merged**